Npgsql.sln

```diff
@@ -29,6 +29,13 @@ Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Npgsql.GeoJSON", "src\Npgsq
 EndProject
 Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Npgsql.Specification.Tests", "test\Npgsql.Specification.Tests\Npgsql.Specification.Tests.csproj", "{A77E5FAF-D775-4AB4-8846-8965C2104E60}"
 EndProject
+Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "Solution Items", "Solution Items", "{004A2E0F-D34A-44D4-8DF0-D2BC63B57073}"
+ProjectSection(SolutionItems) = preProject
+	Directory.Build.props = Directory.Build.props
+	Directory.Build.targets = Directory.Build.targets
+	.github\workflows\build.yml = .github\workflows\build.yml
+EndProjectSection
+EndProject
```

Comment:
Could we avoid this?

Reply:
Why does it bother you so much, just keep the node closed :) These files really are a part of the solution (i.e. they participate in the build), and there's no easy way of editing them without starting to switch to filesystem view etc...

Reply:
Removing them doesn't remove them from the solution. That's the problem :(

Reply:
Then we'll just have to remember to remove them from there :) If we forget to, it's not the end of the world...

BTW projects are the same - you have to remove them both from the filesystem and from the solution (although it's true that projects and individual files aren't exactly the same - mainly trolling :trollface: )